manifests/extension.pp

```diff
@@ -71,6 +78,7 @@
   $settings_prefix   = false,
   $sapi              = 'ALL',
   $responsefile      = undef,
+  $install_options   = undef,
```

Comment:
please add a datatype at least for the parameter (every other parameter is a nice bonus, but not required)

Reply:
can you add a unit or acceptance test for the new parameter?